### PR TITLE
Update the docs building workflow for PPO

### PR DESCRIPTION
**Pull request by varunarora**

### CODE REVIEW

File: scripts/deploy_docs_on_travis.sh [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+export DEPLOY_DOCS_SH=https://raw.githubusercontent.com/PaddlePaddle/PaddlePaddle.org/$PPO_SCRIPT_BRANCH/scripts/deploy/deploy_docs.sh
+
+docker run -it \
+    -e CONTENT_DEC_PASSWD=$CONTENT_DEC_PASSWD \
+    -e TRAVIS_BRANCH=$TRAVIS_BRANCH \
+    -e DEPLOY_DOCS_SH=$DEPLOY_DOCS_SH \
+    -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST \
+    -e PPO_SCRIPT_BRANCH=$PPO_SCRIPT_BRANCH \
+    -e PADDLE_ROOT=/VisualDL \
+    -e PYTHONPATH=/FluidDoc/external/Paddle/build/python \
```
Comment (jetfuel) [inline scripts/deploy_docs_on_travis.sh]: This might be a copy paste error.

Comment (varunarora) [reply scripts/deploy_docs_on_travis.sh]: What's the error?

Comment (varunarora) [reply scripts/deploy_docs_on_travis.sh]: Oh wait that path isn't right?

Comment (jetfuel) [reply scripts/deploy_docs_on_travis.sh]: The path points to `FluidDoc/external`, which we are not using

**Review by jetfuel — DISMISSED** [review]
LGTM

**Review by jetfuel — DISMISSED** [review]

**Review by jetfuel — APPROVED** [review]

**Pull request merged**